gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/map/Substring.feature

```diff
@@ -55,11 +55,11 @@ Feature: Step - substring()
       | p |
       | pple |
 
```

Comment:
General comment, I would add some feature test cases with combinations of (pos start index, neg end index), (neg start index, pos end index), and (neg start index, neg end index).